minecraft_servers/Varies/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "server_name": "Varies",
+  "nice_name": "Varies",
+  "direct_ip": "Play.Varies.Ae",
+  "server_wildcards":[
+     "%.Varies.ae"
+  ],
+  "social": {
+    "web": "https://Varies.ae",
+    "web_shop": "https://Varies.ae/store",
```

Comment:
Can't access website; if non-existent, remove

Reply:
It is there but in maintenance mode right now